services/src/main/java/com/linkedin/kmf/services/TopicManagementService.java

```diff
@@ -209,6 +214,12 @@ public void run() {
     }
   }
 
+  private class TopicManagementServiceThreadFactory implements ThreadFactory {
+    public Thread newThread(Runnable r) {
+      return new Thread(r, _serviceName + " topic-management-service thread");
```

Comment:
Can we also replace space with `-`? I think by tradition we don't use space in the any name -- otherwise we may think the words printed before/after the name in the log is part of the name.